lighthouse-core/scripts/dogfood-lhci.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+set -euox pipefail
+
+# This script requires LHCI_CANARY_SERVER_URL and LHCI_CANARY_SERVER_TOKEN variables to be set.
```

Comment:
> LHCI_CANARY_SERVER_TOKEN

does this mean only PRs from core contributors will get this?

Reply:
I could've sworn I replied to this already...

anyhoo, yes it's only available to core contributors at the moment, but that's not a hard requirement, we could loosen this and hardcode them if we wanted later they're not really *that* secret since you could find them out yourself given the URL of our server.